system-test/partition-testcases/gce-5-node-3-partition.yml

```diff
@@ -0,0 +1,22 @@
+steps:
+  - command: "system-test/testnet-automation.sh"
+    label: "GCE - CPU Only 5 Node - Partial Loss 3 Partitions"
```

Comment:
Can we run this test on colo too?

Reply:
Yes, see the `colo-3-partitions.yml` file.  Each .yml file refers to a single test case configuration, ie, a specific provider, duration, number of nodes, and in this case, what kind of partitions to hit it with.

Reply:
Sorry I guess what I meant was, "let's not run this on GCE and instead use the colo hardware"

Reply:
keep the burn down